deploy/docker/jupyterhub/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:18.04
+FROM frolvlad/alpine-miniconda3
```

Comment:
Can we lock this to a specific version instead of latest?

Reply:
✅